lib/storage/file.js

```diff
@@ -1061,7 +1061,19 @@ File.prototype.getSignedPolicy = function(options, callback) {
 
   makeAuthorizedRequest_.getCredentials(function(err, credentials) {
     if (err) {
-      callback(err);
+      var signingError = new Error('Signing failed. See `error` property.');
+      signingError.error = err;
+      callback(signingError);
```

Comment:
I think this is a case where that would be more engineering than its worth. Sub-classing an Error is quite a noisy process. However, it would be nice to go over our errors sometime with something like https://github.com/floatdrop/create-error-class and maybe lump them into a "lib/common/errors.js" file or even one for each module: "lib/storage/errors.js". Again though, I'm not even sure *that's* worth the effort.